Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_User.c

```diff
@@ -92,7 +93,9 @@ void bsg_kscrashsentry_reportUserException(const char *name, const char *reason,
         bsg_g_context->userException.state = appState;
 
         BSG_KSLOG_DEBUG("Calling main crash handler.");
-        bsg_g_context->onCrash();
+        char errorClass[21];
+        strncpy(errorClass, bsg_g_context->userException.name, sizeof(errorClass));
+        bsg_g_context->onCrash(severity[0], errorClass);
```

Comment:
Can we trust that `severity` will always be a valid value here?

Reply:
Yes, but I added a check for an uncertain future.